privaterelay/templates/socialaccount/authentication_error.html

```diff
@@ -27,6 +27,7 @@
                 <form class="join-waitlist-form flx jst-cntr" id="join-waitlist-form" method="post" action="{% url 'waitlist' %}">
                   {% csrf_token %}
                   <input type="hidden" name="email" type="hidden" value="{{ request.session.waitlist_email }}">
+                  <input type="hidden" name="fxa_uid" type="hidden" value="{{ request.session.waitlist_fxa_uid }}">
```

Comment:
Need to update the JS [here](https://github.com/mozilla/fx-private-relay/blob/master/static/js/app.js#L164) to grab the `fxa_uid` field and send the post.

Reply:
Fixed.